doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -104,118 +132,161 @@ PyROS Solver Interface
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.BudgetSet
+    :show-inheritance:
     :special-members: coefficients_mat, rhs_vec, origin, budget_membership_mat, budget_rhs_vec, type, parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.FactorModelSet
+    :show-inheritance:
     :special-members: origin, number_of_factors, psi_mat, beta, type, parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.PolyhedralSet
+    :show-inheritance:
     :special-members: coefficients_mat, rhs_vec, type, parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.AxisAlignedEllipsoidalSet
+    :show-inheritance:
     :special-members: center, half_lengths, type, parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.EllipsoidalSet
+    :show-inheritance:
     :special-members: center, shape_matrix, scale, type, parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.UncertaintySet
+    :show-inheritance:
     :special-members: parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.DiscreteScenarioSet
+    :show-inheritance:
     :special-members: scenarios, type, parameter_bounds, dim, point_in_set
 
 .. autoclass:: pyomo.contrib.pyros.uncertainty_sets.IntersectionSet
+    :show-inheritance:
     :special-members: all_sets, type, parameter_bounds, dim, point_in_set
 
 
 PyROS Usage Example
 -----------------------------
 
-We will use an example to illustrate the usage of PyROS. The problem we will use is called *hydro* and comes from the GAMS example problem database in `The GAMS Model Library <https://www.gams.com/latest/gamslib_ml/libhtml/>`_. The model was converted to Pyomo format via the `GAMS Convert tool <https://www.gams.com/latest/docs/S_CONVERT.html>`_.
-
-This model is a QCQP with 31 variables. Of these variables, 13 represent degrees of freedom, with the additional 18 being state variables.
-The model features 6 linear inequality constraints, 6 linear equality constraints, 6 non-linear (quadratic) equalities, and a quadratic objective.
-We have augmented this model by converting one objective coefficient, two constraint coefficients, and one constraint right-hand side into Param objects so that they can be considered uncertain later on.
+In this section, we illustrate the usage of PyROS with a modeling example.
+The deterministic problem of interest is called *hydro* 
+(available `here <https://www.gams.com/latest/gamslib_ml/libhtml/gamslib_hydro.html>`_),
+a QCQP taken from the
+`GAMS Model Library <https://www.gams.com/latest/gamslib_ml/libhtml/>`_.
+We have converted the model to Pyomo format using the
+`GAMS Convert tool <https://www.gams.com/latest/docs/S_CONVERT.html>`_.
+
+The *hydro* model features 31 variables,
+of which there are 13 are degrees of freedom and 18 state variables.
```

Comment:
"of which there are 13 are degrees of freedom and 18 state variables" -> "of which 13 are degrees of freedom and 18 are state variables"